src/main/java/org/terasology/joshariasSurvival/systems/SheepShearingSystem.java

```diff
@@ -0,0 +1,78 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.joshariasSurvival.systems;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.terasology.engine.core.Time;
+import org.terasology.engine.entitySystem.entity.EntityManager;
+import org.terasology.engine.entitySystem.entity.EntityRef;
+import org.terasology.engine.entitySystem.event.ReceiveEvent;
+import org.terasology.engine.entitySystem.systems.BaseComponentSystem;
+import org.terasology.engine.entitySystem.systems.RegisterSystem;
+import org.terasology.engine.logic.characters.events.AttackEvent;
+import org.terasology.wildAnimals.event.AnimalSpawnEvent;
+import org.terasology.engine.logic.delay.DelayManager;
+import org.terasology.engine.logic.delay.PeriodicActionTriggeredEvent;
+import org.terasology.engine.logic.inventory.ItemComponent;
+import org.terasology.engine.registry.In;
+import org.terasology.joshariasSurvival.components.ShearableSheepComponent;
+
+
+import java.util.ArrayList;
+
+@RegisterSystem
+public class SheepShearingSystem extends BaseComponentSystem {
+    public static final int HITS_TO_SHEAR = 5;
+    public static final ArrayList<ItemComponent> SHEARING_ITEMS = null; //TODO yet to decide
+    public static final int HAIR_REGROWTH_TIME = 3*60*1000; // In ms
+    public static final String HAIR_REGROWTH_ACTION_ID = "Check hair regrowth";
```

Comment:
ID's should ideally use `Name` and IMO rather be something more technical than "Check hair regrowth"... aka a real _identifier_ rather than a bullet point. I'd recommend something along the lines of `<module>:<action>` to be in line with our asset IDs etc.